### PR TITLE
[2.5] Made help information of commands more consistent

### PR DESCRIPTION
**Pull request by wouterj**
Follow up of https://github.com/symfony/symfony/pull/13231 for commands added in 2.4 and 2.5

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ protected function configure()
 The <info>%command.name%</info> displays the configured routes:
 
   <info>php %command.full_name%</info>
-
+
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php]: Is this change needed?

**Comment by fabpot** [comment]
Thank you @WouterJ.
